mathesar_ui/src/App.svelte

```diff
@@ -141,6 +144,10 @@
     height: 100vh;
   }
 
+  p {
+    line-height: 1.5;
+  }
```

Comment:
This seems a little odd to me. I'm fine with changing the line height. But I wouldn't expect us to customize it specifically for paragraphs. We don't even use `<p>` very often in Mathesar. So I'm worried that this CSS will lead to inconsistent style.

Can you explain why you set this here?

I'd typically expect to see `line-height` set on a very high-level element like `html` and then for it to sometimes be overridden for elements like headings.

Reply:
The blocks of text were too tight and I was concerned about making this a global change in case it affected buttons or other elements like the pills. I will consider your feedback and try adjusting it globally.